deploy/template.yaml

```diff
@@ -1804,6 +1804,13 @@ Resources:
               Ref: IPVCorePrivateAPI
             Path: /journey/build-proven-user-identity-details
             Method: GET
+        IPVCorePrivateAPINewPath:
+          Type: Api
+          Properties:
+            RestApiId:
+              Ref: IPVCorePrivateAPI
+            Path: /user/build-proven-user-identity-details
```

Comment:
The ticket suggests the new path be `/user/proven-user-identity-details`, since from the frontend's POV this is accessing a resource, not an action.

I'd actually suggest simplifying even further to just `/user/proven-identity-details` - I don't see any reason to repeat the user.

Reply:
Done. changed url with '/user/proven-identity-details'